samples/usb/main.c

```diff
@@ -1,39 +1,39 @@
-#include <hal/debug.h>
-#include <hal/video.h>
-#include <windows.h>

```

Comment:
Something's not right with this diff. Maybe an accidental CRLF<->LF change?

Reply:
Looks fine to me, all that's replaced is `hal/debug.h` with `nxdk/log_console.h`

Reply:
No, the diff replaces every single line in the whole file.